src/Symfony/Component/Cache/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * added max-items + LRU + max-lifetime capabilities to `ArrayCache`
  * added `CouchbaseBucketAdapter`
+ * added `IdentityMarshaller`
```

Comment:
this should go next to `MarshallingSessionHandler` to me, not in Cache

Reply:
done